tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -119,30 +127,23 @@ public void testInvalidTomlFile() throws BallerinaTestException {
         errorLeecher4.waitForText(5000);
     }
 
-    @DataProvider(name = "negative-tests")
-    public Object[][] getNegativeTestDetails() {
+    @DataProvider(name = "negative-projects")
+    public Object[][] getNegativeTestProjects() {
         return new Object[][]{
-                {"invalidOrgName", "Value not provided for required configurable variable 'booleanVar'" },
-                {"requiredNegative", "Value not provided for required configurable variable 'stringVar'"},
-                {"noModuleConfig", "Value not provided for required configurable variable 'intVar'"},
-                {"invalidRecordField", "Configurable feature is yet to be supported for field type " +
-                        "'string[][]' in variable 'testUser' of record 'main:AuthInfo'"},
-                {"additionalField", "Additional field 'scopes' provided for configurable variable 'testUser' of " +
-                        "record 'main:AuthInfo' is not supported"},
-                {"missingRequiredField", "Value not provided for non-defaultable required field " +
-                        "'username' of record 'main:AuthInfo' in configurable variable 'testUser'"},
-                {"missingTableKey", "Value required for key 'username' of type " +
-                        "'table<(main:AuthInfo & readonly)> key(username) & readonly' in configurable variable " +
-                        "'users'"},
-                {"invalidMapType", "Configurable feature is yet to be supported for type 'map<int> & readonly'"},
-                {"invalidType", errorMsg + "invalid type found for variable 'intVar', expected type is 'int', found " +
-                        "'DOUBLE'"},
-                {"invalidByteRange", "Value provided for byte variable 'byteVar' is out of range. Expected " +
-                        "range is (0-255), found '355'"}
+                {"invalidComplexArray", "configurable feature is yet to be supported for type " +
```

Comment:
Shall we create issues for types that can be supported as per spec

Reply:
Also not supported cases should be fixed by the front end side. If it is not supported by spec, then should be a compilation. We need to create issues on the compiler front end.

Reply:
created #28976

Reply:
And for the not supported type `map<anydata>` the spec says it is only unsupported in TOML syntax.
https://github.com/ballerina-platform/ballerina-spec/blob/master/configurable/spec.md#configurable-variables
Then, I suppose we shouldn't do the check at compiler FE.